packages/expect/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "jest-matchers",
+  "name": "expect",
```

Comment:
Isn't the plan to *use* expect directly, not to replace it with the contents of jest-matchers? expect has lots of important tests and behaviors that jest-matchers may not be replicating, so semver-major or not, that could end up screwing over a lot of people.

Reply:
@ljharb The plan was always to donate the `expect` package to the Jest team so they could run with it. They are doing an excellent job making an assertion library (jest-matchers) with very similar goals to expect, not to mention the syntax for most common assertions is practically identical, and I'd rather unite the community under one package than two.

As for "screwing over a lot of people" I don't think that's happening here. I've been very clear about my intentions for the expect project for a long time now and most people I talk to seem very supportive of the idea.

Reply:
Fair; I suppose I was just under the wrong impression.

I'm supportive of a package remaining maintained; if it needs to be replaced with a different implementation to do so, that's fine; I just hope that the breaking changes will be minimal and clearly documented.

Reply:
There's a a codemod (https://github.com/skovhus/jest-codemods/issues/62) which will port most of existent `expect` code. I don't think there's any breaking changes beyond that

Reply:
Codemods are great; but I don't think automated ones should be required to be able to have a smooth upgrade experience.